list.go

```diff
@@ -9,15 +9,17 @@ func List(c *cli.Context) error {
 	client := GetClient(c)
 
 	colorList := ColorList()
-	var projectIds []int
-	for _, project := range client.Store.Projects {
-		projectIds = append(projectIds, project.GetID())
+	projectsCount := len(client.Store.Projects)
+	projectIds := make([]int, projectsCount, projectsCount)
+	for i, project := range client.Store.Projects {
+		projectIds[i] = project.GetID()
 	}
 	projectColorHash := GenerateColorHash(projectIds, colorList)
 	ex := Filter(c.String("filter"))
 
-	itemList := [][]string{}
-	for _, itemOrder := range client.Store.ItemOrders {
+  orderItemsCount := len(client.Store.ItemOrders)
+	itemList := make([][]string, orderItemsCount, orderItemsCount)
```

Comment:
`orderItemCount` is not enough.
It should filter out checked task and tasks which rejected by filter option.